src/Umbraco.Tests/IO/ShadowFileSystemTests.cs

```diff
@@ -59,8 +59,8 @@ public void ShadowDeleteDirectory()
             Directory.CreateDirectory(path + "/ShadowTests");
             Directory.CreateDirectory(path + "/ShadowSystem");
 
-            var fs = new PhysicalFileSystem(path + "/ShadowTests/", "ignore", ioHelper);
-            var sfs = new PhysicalFileSystem(path + "/ShadowSystem/", "ignore", ioHelper);
+            var fs = new PhysicalFileSystem(ioHelper, Current.Logger, path + "/ShadowTests/", "ignore");
+            var sfs = new PhysicalFileSystem(ioHelper, Current.Logger, path + "/ShadowSystem/", "ignore");
```

Comment:
Similar to above `Current` isn't initialized here so not sure how this is working, should just be Mock<ILogger>()